dir.traversal.targets

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.targets" />
-
```

Comment:
Unless I'm missing something, I like what I have here better. I don't see the point of that import, and even less the point of moving things around to serve it.